recipes/gpcam/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "gpcam" %}
+{% set version = "7.4.8" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/lbl-camera/gpCAM/archive/{{ version }}.tar.gz
+  sha256: 765ef7acc1563018febd62ccf66499b813db6b0eda102dd2331779c1e91309da
+  patches:
+    - zmq.patch
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+  run:
+    - python >=3.7
+    - wheel
```

Comment:
Wheel is probably not needed.

Reply:
@ocefpaf I added the required patch to remove it from the dependencies.

Reply:
If you can, file an issue upstream too.